src/Dinkbit/Notifyme/Gateways/Twilio.php

```diff
@@ -140,11 +140,11 @@ protected function parseResponse($body)
      */
     protected function responseError($rawResponse)
     {
-        if (! $this->isJson($rawResponse->getBody())) {
-            return $this->jsonError($rawResponse);
+        if ($json = $this->isJson($rawResponse->getBody())) {
```

Comment:
Asignment inside if statements is generally regarded as bad. Symfony doesn't allow it for example.